content/id/docs/contribute/localization_id.md

```diff
@@ -0,0 +1,178 @@
+pencapaian atau _milestone_ dalam kurun waktu tersebut [jadwal rapat daring 
+reguler tim Bahasa Indonesia](https://zoom.us/j/6072809193) dilakukan secara 
+konsisten setiap dua minggu sekali. Dalam [agenda rapat ini](https://docs.google.com/document/d/1Qrj-WUAMA11V6KmcfxJsXcPeWwMbFsyBGV4RGbrSRXY) 
+juga dilakukan pemilihan PR _Wrangler_ untuk dua minggu ke depan. Tugas PR 
+_Wrangler_ tim Bahasa Indonesia serupa dengan PR _Wrangler_ dari proyek 
+_upstream_.
+
+Target pencapaian atau _milestone_ tim akan dirilis sebagai 
+[_issue tracking_ seperti ini](https://github.com/kubernetes/website/issues/22296) 
+pada Kubernetes GitHub Website setiap 4 bulan. Dan bersama dengan informasi 
+PR _Wrangler_ yang dipilih setiap dua minggu, keduanya akan diumumkan di Slack 
+_channel_ [#kubernetes-docs-id](https://kubernetes.slack.com/archives/CJ1LUCUHM) 
+dari Komunitas Kubernetes.
+
+## Cara Memulai Translasi
+
+Untuk menerjemahkan satu halaman Bahasa Inggris ke Bahasa Indonesia, lakukan 
+langkah-langkah berikut ini:
+
+* Check halaman _issue_ di GitHub dan pastikan tidak ada orang lain yang sudah 
+mengklaim halaman kamu dalam daftar periksa atau komentar-komentar sebelumnya.
+* Klaim halaman kamu pada _issue_ di GitHub dengan memberikan komentar di bawah 
+dengan nama halaman yang ingin kamu terjemahkan dan ambillah hanya satu halaman 
+dalam satu waktu.
+* _Fork_ [repo ini](https://github.com/kubernetes/website), buat terjemahan 
+kamu, dan kirimkan PR (_pull request_) dengan label `language/id`
+* Setelah dikirim, pengulas akan memberikan komentar dalam beberapa hari, dan 
+tolong untuk menjawab semua komentar. Direkomendasikan juga untuk melakukan 
+[_squash_](https://github.com/wprig/wprig/wiki/How-to-squash-commits) _commit_ 
+kamu dengan pesan _commit_ yang baik.
+
+
+## Informasi Acuan Untuk Translasi
+
+Tidak ada panduan gaya khusus untuk menulis translasi ke bahasa Indonesia. 
+Namun, secara umum kita dapat mengikuti panduan gaya bahasa Inggris dengan 
+beberapa tambahan untuk kata-kata impor yang dicetak miring.
+
+Harap berkomitmen dengan terjemahan kamu dan pada saat kamu mendapatkan komentar
+dari pengulas, silahkan atasi sebaik-baiknya. Kami berharap halaman yang 
+diklaim akan diterjemahkan dalam waktu kurang lebih dua minggu. Jika ternyata 
+kamu tidak dapat berkomitmen lagi, beri tahu para pengulas agar mereka dapat 
+meberikan halaman tersebut ke orang lain.
+
+Beberapa acuan tambahan dalam melakukan translasi silahkan lihat informasi 
+berikut ini:
+
+### Daftara Glosarium Translasi dari tim SIG DOC Indonesia
+Untuk kata-kata selengkapnya silahkan baca glosariumnya 
+[disini](#glosarium-indonesia)
+
+### KBBI
+Konsultasikan dengan KBBI (Kamus Besar Bahasa Indonesia) 
+[disini](https://kbbi.web.id/) dari 
+[Kemendikbud](https://kbbi.kemdikbud.go.id/).
+
+### RSNI Glosarium dari Ivan Lanin
+[RSNI Glosarium](https://github.com/jk8s/sig-docs-id-localization-how-tos/blob/master/resources/RSNI-glossarium.pdf)
```

Comment:
is it possible to commit this PDF to k/website under our ID folder and link it to this instead?

Reply:
How about put in the `id/_common-resources/` @girikuncoro?